src/Core/p2/iSnd.cpp

```diff
@@ -96,28 +168,80 @@
 #pragma GLOBAL_ASM("asm/Core/p2/iSnd.s", "iSndSetPitch__FUif")
 
 // func_800D2560
-#pragma GLOBAL_ASM("asm/Core/p2/iSnd.s", "iSndStartStereo__FUiUif")
+void iSndStartStereo()
+{
+}
 
 // func_800D2564
-#pragma GLOBAL_ASM("asm/Core/p2/iSnd.s", "iSndStereo__FUi")
-
+void iSndStereo(uint32 i)
+{
+    if (i == 0)
+    {
+        OSSetSoundMode(0);
+        gSnd.stereo = 0;
+    }
+    else
+    {
+        OSSetSoundMode(1);
+        gSnd.stereo = 1;
+    }
+}
+
+#ifndef NON_MATCHING
 // func_800D25B4
 #pragma GLOBAL_ASM("asm/Core/p2/iSnd.s", "iSndWaitForDeadSounds__Fv")
+#else
+void iSndWaitForDeadSounds()
+{
+    fc = 0;
+    int i = 0;
+    // Can't get 0x8c to get stored in r31
+    while ((i = fc) < 0x8c) //for (int i = 0; (i = fc) < 0x8c; )
+    {
+        i = fc;
+        while (fc < i + 0xe)
+            ;
```

Comment:
Idk what's going on with this syntax here, did you run clang-format on this file? this looks weird.

Reply:
yeah it just won't keep the semicolon there. I meant to add a set of empty braces to make it look nicer but I forgot

Reply:
actually braces make it worse lol
```
while (fc < i + 0xe)
        {
        };
```

Reply:
It just looked funny to me, it's fine as it is, I just wanted to make sure you ran the formatting on the file.

Reply:
I have it enabled on save now